test/e2e/networking.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Networking", func() {
 					{
 						Name:    contName,
 						Image:   "gcr.io/google_containers/busybox",
-						Command: []string{"ping", "-c", "3", "-w", "10", "google.com"},
+						Command: []string{"nc", "-w", "30", "google.com", "80"},
```

Comment:
I think that if you use the -z flag, works without a TTY. But I don't feel strongly either way.  wget or curl are also fine.

  -z      Specifies that nc should just scan for listening daemons, without sending any data to them.  It is an error to use this option in
             conjunction with the -l option.